build.gradle

```diff
@@ -1,18 +1,31 @@
 plugins {
     id 'nebula.netflixoss' version '2.2.5'
     id 'nebula.provided-base' version '2.0.1'
+    id "net.linguica.maven-settings" version "0.5"
 }
 
 ext.githubProjectName = rootProject.name // Change if github project name is not the same as the root project's name
 
+license {
+    ignoreFailures = true
+}
+
 subprojects {
     apply plugin: 'nebula.netflixoss'
     apply plugin: 'java'
+    apply plugin: 'net.linguica.maven-settings'
 
     group = "com.netflix.${githubProjectName}" // TEMPLATE: Set to organization of project
 
     repositories {
-        jcenter()
+        if (project.hasProperty("repoName") && project.hasProperty("repoUrl")) {
```

Comment:
The credentials will be read from the maven's `settings.xml` file. Specifically, the value of `repoName` should points to one of the `id` tag under `settings->servers->server`, then the plugin should use the corresponding crendentials